src/main.rs

```diff
@@ -107,8 +107,12 @@ pub fn main() -> Result {
     });
     runtime.shutdown_timeout(Duration::from_secs(0));
 
+    if let Err(e) = &res {
+        let error_logger = slog_scope::logger().new(o!());
+        error!(&error_logger, "{e}");
```

Comment:
I think you can just use `logger` above? No need to create yet another scope.

Reply:
I believe you can just use `logger` or worst case clone it.. no need to re-scope one